src/sp_disabled_functions.c

```diff
@@ -271,24 +282,23 @@ bool should_disable_ht(zend_execute_data* execute_data,
 
   ht_entry = zend_hash_str_find_ptr(ht, function_name, strlen(function_name));
 
-  if (ht_entry &&
-      should_disable(execute_data, function_name, builtin_param,
-                     builtin_param_name, ht_entry, current_filename)) {
-    ret = true;
+  if (ht_entry) {
```

Comment:
I changes the execution flow if ht_entry is true and first should disable fails. Previously the second attempt (if config is true) could succeed, but not anymore. Is it intended ?

Reply:
If the function is in the HashTable `ht`, there is no point in looking in `config`, is it?

Remember that thanks to @xXx-caillou-xXx the functions that should be disabled are split into two hashtable: one containing functions that SP is able to hook at startup time, and the second for the others. There is no intersection between the two HashTable :)